packages/client/hmi-client/src/utils/pivot.ts

```diff
@@ -104,36 +103,53 @@ export const createMatrix1D = (data: any[]) => {
 		rows.push(row);
 	}
 
-	return { matrix: rows };
+	return rows;
```

Comment:
??? I don't understand the change here - are you even using this function??

Reply:
Thought it made it less complicated when assigning a value to `createMatrix1D`. I'll revert it to keep things consistent